modules/plugins/postgres/src/main/java/org/rhq/plugins/postgres/PostgresDatabaseComponent.java

```diff
@@ -342,4 +346,80 @@ private OperationResult invokeSql(Configuration parameters) throws SQLException
         }
     }
 
+    /**
+     * This functionality is a port of vacuumlo tool in the Postgres distribution
+     *
+     * @return OperationResult
+     */
+    private OperationResult vacuumLo() {
+        Connection c = null;
+        Statement clearVacuumL = null;
+        Statement unlinkStatement = null;
+        ResultSet rs = null;
+        try {
+            c = getPooledConnectionProvider().getPooledConnection();
+            PreparedStatement tempTable = c.prepareStatement("CREATE TEMP TABLE vacuum_l AS (SELECT DISTINCT loid AS lo FROM pg_largeobject)");
```

Comment:
Well, it works with 9.5.5 at least, if the user owns the database in question (I have rhqadmin user that is the owner of rhqdev database).

Do we have a use-case where the rhq-user has access to modify the schema, but not to access pg_largeobject ?